Gemfile

```diff
@@ -5,13 +5,24 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+
+group :test do
+  gem 'database_cleaner'
+  gem 'faker'
+  gem 'shoulda-matchers', '~> 3.1'
+end
+group :development, :test do
```

Comment:
This group it's repetead.

The gemfile should have this format:

- Global gems.
- Shared groups of environments.
- Individual environments.